android/app/src/googleplay/java/org/fossasia/openevent/utils/map/ImageUtils.java

```diff
@@ -0,0 +1,31 @@
+package org.fossasia.openevent.utils.map;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.graphics.drawable.Drawable;
+import android.support.annotation.DrawableRes;
+import android.support.v4.content.ContextCompat;
+import android.support.v4.content.res.ResourcesCompat;
+import android.support.v4.graphics.drawable.DrawableCompat;
+
+import com.google.android.gms.maps.model.BitmapDescriptor;
+import com.google.android.gms.maps.model.BitmapDescriptorFactory;
+
+/**
+ * Created by rohanagarwal94 on 6/7/17.
+ */
+public class ImageUtils {
+
+    public static BitmapDescriptor vectorToBitmap(Context context, @DrawableRes int id, int color) {
+        Drawable vectorDrawable = ResourcesCompat.getDrawable(context.getResources(), id, null);
+        assert vectorDrawable != null;
```

Comment:
to make sure the vector drawable is not null. Otherwise would have to check it at every implementation of the method. Any better way in your mind?